src/app/public/test-utility/test-utility.ts

```diff
@@ -146,4 +146,49 @@ export class SkyAppTestUtility {
     return skyEl;
   }
 
+  /**
+   * Returns the first element inside the overlay that is a match of the
+   * provided selector. Selector should not contain SKYUX prefixes, as they are for
+   * internal use only and are subject to change.
+   */
+  public static overlayQuerySelector(selector: string): any {
+    if (selector.substring(0, 5) === '.sky-' || selector.substring(0, 4) === 'sky-') {
```

Comment:
What about selectors that don't start with `.sky-`? Such as `#foobar .sky-colorpicker`? You could just do an `indexOf` check...

Reply:
Good call.

Reply:
Fixed.